1-exercises/A-reading-list/script.js

```diff
@@ -1,27 +1,52 @@
+const content = document.getElementById("content");
+content.className = "navbar-brand alert";
+content.style.alignItems = "center";
+//💫💫💫
 function readingList(books) {
+  books.forEach((e) => {
+    let ul = document.createElement("ul");
```

Comment:
You should create the unordered list outside of the for loop, and add the background colour to the list items.